src/vcSettings.cpp

```diff
@@ -209,6 +209,12 @@ bool vcSettings_Load(vcSettings *pSettings, bool forceReset /*= false*/, vcSetti
     pSettings->camera.lensIndex = data.Get("camera.lensId").AsInt(vcLS_30mm);
     pSettings->camera.fieldOfView = data.Get("camera.fieldOfView").AsFloat(vcLens30mm);
 
+    pSettings->displacementShaderType = (vcDisplacementShaderType)data.Get("displacementShaderType").AsInt(0);
+    pSettings->visualization.displacement.bounds.x = data.Get("visualization.displacement.minBound").AsFloat(0.f);
+    pSettings->visualization.displacement.bounds.y = data.Get("visualization.displacement.maxBound").AsFloat(1.f);
+    pSettings->visualization.displacement.errorColour = data.Get("visualization.displacement.errorColour").AsInt(0);
+    pSettings->visualization.displacement.outOfBoundsColour = data.Get("visualization.displacement.outOfBoundsColour").AsInt(16777215);
```

Comment:
I'd prefer to see colours like this specified in hex instead, `0xFFFFFF`. Makes it much clearer what kind of colour you're going for.